### PR TITLE
Add tos library of some popular services

### LINKED ISSUE #6: As a user, I should be able to choose from common TOS to add to my form and submit it. (so the user doesn’t need to go find the TOS on their own)

### PR DESCRIPTION
**Pull request by kirch1**
### Files Changed:
- Process.tsx
- Process.css
### What Has Changed: 
- Add a list of some popular services to get the user started
### Closes: (any issues that it closes)
Closes #6

### CODE REVIEW

File: src/Components/Process/Process.tsx [hunk 1]
```diff
@@ -53,11 +71,23 @@ interface FormProps {
...
+  const handleTOSChange = (tos: string) => {
+    setSelectedLibrary('');
+    setTosInput(tos);
+  }
+
+  const handleLibraryChange = (event: SelectChangeEvent) => {
+    setSelectedLibrary(event.target.value);
+    setTosInput(event.target.value);
+  };
+
```
Comment (Jesusborjas006) [inline src/Components/Process/Process.tsx:90]: Great job setting state to whichever value is selected.

File: src/tosLibrary.tsx
Comment (Jesusborjas006) [inline src/tosLibrary.tsx:1]: Nice way of storing different TOS's!

**Review by Jesusborjas006 — APPROVED** [review]

File: src/Components/Process/Process.css [hunk 2]
```diff
@@ -28,7 +28,7 @@
   min-width: 300px;
   display: flex;
   flex-direction: column;
-  gap: 1rem;
+  gap: 1.2rem;
```
Comment (chrissycooper) [inline src/Components/Process/Process.css:31]: I always appreciate this level of attention to detail. TY

File: src/Components/Process/Process.tsx [hunk 3]
```diff
@@ -17,34 +28,49 @@ interface FormProps {
...
 
   const history = useHistory();
 
   const sendTOS = async () => {
     try {
       setLoading(true);
-      const concerns = Object.keys(concernAreas).filter(key => concernAreas[key]);
-      const TOSinfo = await processTOS(tosInput.replace('"', "'"), concerns, setError, user);
+      const concerns = Object.keys(concernAreas).filter(
+        (key) => concernAreas[key]
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:49]: nice, I've never seen an iterator formatted like this, but I like it a lot! Very readable.

File: src/Components/Process/Process.tsx [hunk 4]
```diff
@@ -17,34 +28,49 @@ interface FormProps {
...
+        concerns,
+        setError,
+        user
+      );
       setConcerns(TOSinfo.data);
       setLoading(false);
       history.push("/results");
-    } catch (error:any) {
-      const errorMessage:string = error.message;
+    } catch (error: any) {
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:60]: thank you, nice attention to detail!!

File: src/Components/Process/Process.tsx [hunk 5]
```diff
@@ -66,41 +104,52 @@ interface FormProps {
       <form className="form-card">
         {loading ? (
           <>
-            <h3 className="form-heading">
-              Processing Terms of Service, Please Wait
-            </h3>
+            <h3 className="form-heading"> Processing Terms of Service, Please Wait</h3>
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:107]: Ahhhh a space in the front but not one in the back 😵

File: src/Components/Process/Process.tsx [hunk 6]
```diff
@@ -53,11 +79,23 @@ interface FormProps {
         control={<Checkbox size="small" />}
         label={key}
         checked={concernAreas[key]}
```
Comment (chrissycooper) [inline src/Components/Process/Process.tsx:81]: Having these as boolean values is really smooth, I like how it's useful to check if they exist AND to check the box

File: src/Components/Results/Results.tsx [hunk 7]
```diff
@@ -53,6 +53,7 @@ const Results = ({ concerns }: ResultsProps) => {
               <h3 className="result-concern-title">
                 {concerns[selectedConcern].title}
               </h3>
+              <Tooltip title="Consumer Friendliness Rating">
```
Comment (chrissycooper) [inline src/Components/Results/Results.tsx:56]: Oh! I was working on adding a Popover element, but this might be a better choice

**Review by chrissycooper — APPROVED** [review]

**Pull request merged**